packages/message/src/Collections.ts

```diff
@@ -17,6 +17,8 @@ type CollectionInfo = {
   version: string
 };
 
+const getCollectionIdSeparator = () => '___';
```

Comment:
Maybe a variable instead of a function?

Reply:
5594949704c311196374f5473a4d50a28363529d